src/query/grammar.rs

```diff
@@ -217,76 +285,97 @@ pub fn consume_definition<'a, S>(s: &'a str) -> Result<(Definition<'a, S>, &'a s
-                                    items: Vec::new()
-                                },
-                            }),
-                        ],
+                        span: (Pos { line: 1, column: 1 }, Pos { line: 1, column: 5 }),
+                        items: vec![Selection::Field(Field {
+                            position: Pos { line: 1, column: 3 },
+                            alias: None,
+                            name: "a".into(),
+                            arguments: Vec::new(),
+                            directives: Vec::new(),
+                            selection_set: SelectionSet {
+                                span: (Pos { line: 1, column: 3 }, Pos { line: 1, column: 3 }),
+                                items: Vec::new()
+                            },
+                        }),],
                     }
-                ))
-            ],
-        });
+                ))],
+            }
+        );
     }
 
     #[test]
     fn builtin_values() {
-        assert_eq!(ast("{ a(t: true, f: false, n: null) }"),
+        assert_eq!(
+            ast("{ a(t: true, f: false, n: null) }"),
             Document {
-                definitions: vec![
-                    Definition::Operation(OperationDefinition::SelectionSet(
-                        SelectionSet {
-                            span: (Pos { line: 1, column: 1 },
-                                   Pos { line: 1, column: 33 }),
-                            items: vec![
-                                Selection::Field(Field {
-                                    position: Pos { line: 1, column: 3 },
-                                    alias: None,
-                                    name: "a".into(),
-                                    arguments: vec![
-                                        ("t".into(),
-                                            Value::Boolean(true)),
-                                        ("f".into(),
-                                            Value::Boolean(false)),
-                                        ("n".into(),
-                                            Value::Null),
-                                    ],
-                                    directives: Vec::new(),
-                                    selection_set: SelectionSet {
-                                        span: (Pos { line: 1, column: 3 },
-                                               Pos { line: 1, column: 3 }),
-                                        items: Vec::new()
-                                    },
-                                }),
+                definitions: vec![Definition::Operation(OperationDefinition::SelectionSet(
+                    SelectionSet {
+                        span: (
+                            Pos { line: 1, column: 1 },
+                            Pos {
+                                line: 1,
+                                column: 33
+                            }
```

Comment:
This makes it harder to read and grasp the overall structure of the Document.

Reply:
I do strongly disagree with this sentiment. This is so much more readable. It is a pitty that one is structured over two lines and one over a single line, but that is still better than having the first `Pos` on the same line as the opening brace of the tuple.

Reply:
> but that is still better than having the first Pos on the same line as the opening brace of the tuple.

You mean like this code rustfmt'ed above:
https://github.com/graphql-rust/graphql-parser/pull/58/files/40945037f70328dab53b92faf0470b2942f648b9#diff-5a074e90f57e62a2a1b2fbdfa8d08a53c4bc8027fa320ceb3b9c41559bffb8c8R304
(I understand that this is nitpicking, but well)